tests/system_tests_http.py.in

```diff
@@ -250,8 +250,9 @@ def listener(**kwargs):
         self.assert_get("https://localhost:%s" % r.ports[0])
         # requireSsl=false Allows simple-ssl HTTP
 
-        # Commenting out the following assert until DISPATCH-1513 is fixed.
-        #self.assert_get("http://localhost:%s" % r.ports[0])
+        # The following test will be commented out if you are runnung a version lower than
```

Comment:
"runnung"

Reply:
These are compile-time tests that build software with or without some features. Don't we really want code that is compiled to handle both cases and then detect what to do at run-time?

Reply:
We do have c code that is conditionally compiled using an extra flag based on the version of LWS available on the system. The python tests have to be dealt with separately as well, so we test the version in the  cmake/FindLibWebSockets.cmake  file